src/components/Contacts/ContactDetails/ContactDetailsTab/ContactDetailsTab.tsx

```diff
@@ -156,7 +156,6 @@ export const ContactDetailsTab: React.FC<ContactDetailTabProps> = ({
       setDeleteModalOpen(false);
       onClose();
     } catch (error) {
-      enqueueSnackbar(error.message, { variant: 'error' });
       throw error;
     }
```

Comment:
I think you can just remove the catch altogether. Catching and rethrowing doesn't doesn't add anything that the original error being thrown didn't do.